### DISCUSSION TITLE
Deprecation notices: composite, ndi, and sdl

### DISCUSSION THREAD

**Discussion by ddennedy** [mltframework/mlt#1027] [Announcements]
The `composite` transition is being deprecated for offering poor image quality when animating transformation and poor code quality, partly by trying to do too much.
The 'ndi' consumer has not been maintained and rarely used by anyone (e.g. the open source video editors).
The `sdl` consumer is using a very older version 1.2 of SDL that has been superseded by SDL2.

**Comment by bmatherly** [comment]
Don't forget that we will need to update the documentation to change examples using composite to instead use affine.

**Comment by ddennedy** [comment]
Thanks for the reminder. I just pushed an update to the web site for many things. The `demo` directory in the source tree requires more work.
